### PR TITLE
feat: Add support for ROG Ally

### PR DESCRIPTION
**Pull request by EyeCantCU**
Adds support for the Asus ROG Ally

### CODE REVIEW

File: Containerfile
Comment (KyleGospo) [inline Containerfile:1]: also remove vpower, & powerbuttond
sdgyrodsu can stay iirc, it supports ally

Comment (EyeCantCU) [reply Containerfile:1]: Done :)

**Comment by KyleGospo** [comment]
Spoke with @bsherman about this, my understanding is we'll eventually be able to build kmods for this kernel in ublue-os/akmods and then we can get this shipped @EyeCantCU.

**Comment by EyeCantCU** [comment]
> Spoke with @bsherman about this, my understanding is we'll eventually be able to build kmods for this kernel in ublue-os/akmods and then we can get this shipped @EyeCantCU.

Awesome! That would be perfect

**Comment by KyleGospo** [comment]
https://github.com/ublue-os/asus

Follow here for further updates

**Pull request closed without merging**